data_steward/cdr_cleaner/cleaning_rules/deid/motor_vehicle_accident_suppression.py

```diff
@@ -131,7 +131,7 @@ def __init__(self, project_id, dataset_id, sandbox_dataset_id):
             project_id=project_id,
             dataset_id=dataset_id,
             sandbox_dataset_id=sandbox_dataset_id,
-            affected_tables=resources.CDM_TABLES,
+            affected_tables=AOU_REQUIRED,
```

Comment:
The `AOU_REQUIRED` tables are the ones we require sites to submit.  Correct?

Reply:
yes, but these are the main cdm tables that will have concept_id fields.